upgrade/1.5.2/CSM-With-Other-Products.md

```diff
@@ -0,0 +1,278 @@
+# CSM Only Upgrade on a System with Other Products
+
+This page provides guidance for systems with products installed that are performing an upgrade from a CSM `v1.5.X` release to the CSM `v1.5.2` release.
+
+The [`v1.5.2` upgrade page](../1.5.2/README.md) will refer to this page
+during [Update NCN images](../1.5.2/README.md#update-ncn-images).
+
+**If exclusively CSM is installed on the system, return to [Update NCN images](../1.5.2/README.md#update-ncn-images) and
+choose option 2.**
+
+## Requirements
+
+* `CSM_RELEASE_VERSION` is set in the shell environment on `ncn-m001`. See [Preparation](README.md#preparation) for details on how it should be set.
+
+* Set `CSM_RELEASE` in the shell environment to the same value as `CSM_RELEASE_VERSION`.
+
+    ```bash
+   export CSM_RELEASE="${CSM_RELEASE_VERSION}"
+   echo "${CSM_RELEASE}"
+   ```
+
+## Overview
+
+This process creates new NCN node images and CFS configurations in order to acquire any changes from CSM config.
+These images and CFS configurations will be assigned to the NCNs during this process.
+Later in the CSM `v1.5.2` patch process, nodes will be rebooted into these images and the CFS configuration will be applied to the nodes.
+
+## Steps
+
+### Using `sat bootprep` with IUF generated input files
+
+The name of the IUF activity used to install the HPE Cray EX software products must be known before proceeding. See the
+[Activities](../../operations/iuf/IUF.md#activities) section of the IUF documentation for more
+information on IUF activities. See [`list-activities`](../../operations/iuf/IUF.md#list-activities)
+for information about listing the IUF activities on the system. The first step provides an
+example showing how to find the IUF activity.
+
+1. (`ncn-m001#`) Find the IUF activity used for the most recent install of the system.
+
+   ```bash
+   iuf list-activities
+   ```
+
+   This will output a list of IUF activity names. For example, if only a single install has been
+   performed on this system of the 24.01 recipe, the output may show a single line like this:
+
+   ```text
+   24.01-recipe-install
+   ```
+
+1. (`ncn-m001#`) Record the most recent IUF activity name and directory in environment variables.
+
+   ```bash
+   export ACTIVITY_NAME=
+   ```
+
+   ```bash
+   export ACTIVITY_DIR="/etc/cray/upgrade/csm/iuf/${ACTIVITY_NAME}"
+   ```
+
+1. (`ncn-m001#`) Record the media directory used for this activity in an environment variable.
+
+   ```bash
+   export MEDIA_DIR="$(yq r "${ACTIVITY_DIR}/state/stage_hist.yaml" 'summary.media_dir')"
```

Comment:
Not critical, but sharing regardless. Try the `yq` 4 syntax (installed on NCNs since 0.9 or 1.0 as `yq4`).
```suggestion
   export MEDIA_DIR="$(yq4 eval '.summary.media_dir' "${ACTIVITY_DIR}/state/stage_hist.yaml")"
```